src/utils/get-config.ts

```diff
@@ -60,7 +59,7 @@ export default async function getConfig(reconstruct?: true): Promise<Config> {
 	const validate: Validate = {
 		injectAsDevDependency: (value) => typeof value === 'boolean',
 		testProjectsDirectory: (value) => typeof value === 'string' && fs.existsSync(value),
-		yarnMutexFilePath: (value) => typeof value === 'string' && fs.existsSync(value),
+		yarnMutexPort: (value) => typeof value === 'number',
```

Comment:
We also need to check that this is an integer; floats wouldn't be valid.

Reply:
Should I also check if it's a valid port number? (Something like `value > 0 && value < 65536`)

Reply:
Oh, yeah, that'd be good too!